src/components/grid-aware/DonationBlock/DonationBlock.js

```diff
@@ -22,50 +22,96 @@ const LGLForm = () => (
 
 const DonationBlock = ({
   mainTitle,
+  givingTuesdayImage,
+  givingTuesdayParagraph,
   mainDescription,
   whyDonateTitle,
   whyDonateList,
   impactTitle,
   impactList,
-}) => (
-  <div className={s.bleedWrapper}>
-    <div className={s.gridParent}>
-      <div className={s.gridAreaHeadline}>
-        <h1 className={s.mainTitle}>{mainTitle}</h1>
-        <div className={s.mainDescription}>{mainDescription}</div>
-      </div>
-      <div className={s.gridAreaForm}>
-        <LGLForm />
-      </div>
-      <div className={s.gridAreaDescription}>
-        <h2 className={s.whyDonateTitle}>{whyDonateTitle}</h2>
-        <ul className={s.list}>
-          {whyDonateList.map((reason) => (
-            <li className={s.listItem} key={reason}>
-              {reason}
-            </li>
-          ))}
-        </ul>
-        <h2 className={s.impactTitle}>{impactTitle}</h2>
-        <ul className={s.list}>
-          {impactList.map((reason) => (
-            <li className={s.listItem} key={reason}>
-              {reason}
-            </li>
-          ))}
-        </ul>
+}) => {
+  let title;
+
+  if (mainTitle && !givingTuesdayImage) {
+    title = <h1 className={s.mainTitle}>{mainTitle}</h1>;
+  } else if (givingTuesdayImage && !mainTitle) {
+    title = (
+      <img
+        className={s.mainTitleImage}
+        src={givingTuesdayImage.url}
+        alt={givingTuesdayImage.alt}
+      />
+    );
+  } else {
+    throw Error("Please specify only a mainTitle or givingTuesdayImage prop.");
+  }
+
+  return (
+    <div className={s.bleedWrapper}>
+      <div className={s.gridParent}>
+        <div className={s.gridAreaHeadline}>
+          {title}
+          <div className={s.mainDescription}>{mainDescription}</div>
+          {givingTuesdayParagraph &&
+            givingTuesdayParagraph.map((paragraph, index) => (
+              <div
+                className={s.givingTuesdayParagraph}
+                key={`${paragraph + index}`}
+              >
+                {paragraph}
+              </div>
+            ))}
```

Comment:
Let me make a few independent comments about this:

1. If you ever find yourself doing `foo && foo.map(...)`, then you should just give `foo` a default value of the empty array, as that would allow you to just call `foo.map(...)` without the guard because an empty array will map to the empty array anyway.
2. You should never be generating a React [key](https://reactjs.org/docs/lists-and-keys.html#keys) with the iteration index, since that defeats the purpose of the key in the first place, which is to have an identifier that is independent of the index.
3. I wouldn't advise mapping over all of these in the first place. This is one of the rare instances where I think tag selectors would be appropriate, as long as they are scoped by some other ancestor class name.

Could you just directly render these paragraphs, possibly wrapped by a single div with a class name? See @cw323's One-Paragraph Block component for an example of this: https://github.com/ShelterTechSF/sheltertech.org/pull/95

Reply:
> A few last comments that I got from talking to Derek about this just now:
> 
> * He suggested that we make the left area just span 5 columns instead of 6, since the text is a bit too close to the donation form.
> * We should update the actual embedded form itself to use the special Giving Tuesday version of the form. Here's the embed code:
> 
> ```
> <iframe onload="window.parent.scrollTo(0,0)" height="1053" allowTransparency="true" frameborder="0" scrolling="no" style="width:100%;border:none"  src="https://secure.lglforms.com/form_engine/s/0wDrXsUXwzxrT32_RHQGrw"><a href="https://secure.lglforms.com/form_engine/s/0wDrXsUXwzxrT32_RHQGrw">Fill out my LGL Form!</a></iframe>
> ```
> 
> Note that I made a few manual tweaks to the last one, such as increasing the height slightly and enabling scrolling.

Made changes

Reply:
"Could you just directly render these paragraphs, possibly wrapped by a single div with a class name? See @cw323's One-Paragraph Block component for an example of this: #95" I used a parent class and element selector as specified below